infra/prover-cluster/Pulumi.prod-release-8453.yaml

```diff
@@ -34,7 +34,7 @@ config:
     secure: v1:9FrKfdSIsieo1bxD:S8s/9tyJN0U4V2r4PT81CQYWHgR9tbA=
   prover-cluster:apiKey:
     secure: v1:s3dh8LCcaq1tXtT2:Rbsvh1ktdKrD9WkEVc403al+3xRDFbbWp1FSKAlqsFy9t5mHBhTzXSF6hcTn8dzFsO/1+mNUDCwQktQ=
-  prover-cluster:allowClientAddresses: '["0x067D62CBCC72bBb02358cEB85aBE817249AB616b", "0xa6e98d0ba0ca584509ebcbe00bb36f0c5cfcdd1b", "0x418be9167e835ad820df53fd16b8fe02f796ce1c", "0xda824d974d8bac25a0611b7ab5f091d22f63a234", "0x3d089c2f2cb86d4efde153c81cabd4579784430b", "0x323ec32ef13716bfdc3e0b2a96d7bd7cbcb9d57b", "0x2D611BE1e2E49C7b639A88b507b532DaE35e492b", "0xb59bb74fa0c1611eF6A4989a92C0d3ca6942fC0c", "0x1578934C9B006B9568D884C421704d996D402B78", "0x7276fa3c23c1783bdcb97258717f8f32b75919c9"]'
+  prover-cluster:allowClientAddresses: '["0x067D62CBCC72bBb02358cEB85aBE817249AB616b", "0xa6e98d0ba0ca584509ebcbe00bb36f0c5cfcdd1b", "0x418be9167e835ad820df53fd16b8fe02f796ce1c", "0xda824d974d8bac25a0611b7ab5f091d22f63a234", "0x3d089c2f2cb86d4efde153c81cabd4579784430b", "0x323ec32ef13716bfdc3e0b2a96d7bd7cbcb9d57b", "0x2D611BE1e2E49C7b639A88b507b532DaE35e492b", "0xb59bb74fa0c1611eF6A4989a92C0d3ca6942fC0c", "0x1578934C9B006B9568D884C421704d996D402B78", "0x7276fa3c23c1783bdcb97258717f8f32b75919c9", "0xb686f080c2c7045d13d196787c99bc9972480fd3", "0x0d4b7e56a3d78af33a969938724656c886543444"]'
```

Comment:
should these be all lowercase for consistency? IDK what you do is this codebase though